### DISCUSSION TITLE
Welcome to cgft-llm Discussions!

### DISCUSSION THREAD

**Discussion by echonoshy** [echonoshy/cgft-llm#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by colaliu1** [comment]
Hi，作者：
   我正在了解大模型相关的技术，我想跟您请教一下，大模型对图像的判异有没有可以借鉴的项目或者资料。

**Reply by echonoshy** [reply]
我做图像相关的项目并不算太多，您可以参考：https://mp.weixin.qq.com/s?__biz=MzIzOTU0NTQ0MA==&mid=2247547823&idx=1&sn=f33d6ddec7d798e68d7f998d21352dfd&chksm=e8b23cb8a69c470118e97efd9ebded7b9bef13d12243d7e687198e446fa011501f5222bcbd94#rd

阿里的一个解决方案，使用图像相似度 + 大模型内容理解来实现， 希望可以帮到您。 当然解决方案可能还有很多。
